### PR TITLE
Upgrade to Flask-1.1.4

### PR DESCRIPTION
**Pull request by abulte**
Fix https://github.com/etalab/data.gouv.fr/issues/280

### CODE REVIEW

**Comment by abulte** [comment]
TODO: investigate why this is now needed https://github.com/opendatateam/udata/pull/2639/files#diff-8131e08f2cfc9ac2fde1adc967e6cb708affc65b24f7f71750bda9c2c6a4ee24R304, only on users.

File: udata/settings.py [hunk 1]
```diff
@@ -494,6 +494,7 @@ class Testing(object):
         'title^2',
         'description',
     )
+    FS_IMAGES_OPTIMIZE = True
```
Comment (abulte) [inline udata/settings.py:497]: Somehow this has been overwritten from Defaults while migrating deps in this PR.

File: udata/app.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
 
 from flask_wtf.csrf import CSRFProtect
 from speaklater import is_lazy_string
-from werkzeug.contrib.fixers import ProxyFix
+from werkzeug.middleware.proxy_fix import ProxyFix
```
Comment (abulte) [inline udata/app.py:18]: Avoid deprecation warning

**Review by quaxsze — APPROVED** [review]

File: .circleci/config.yml [hunk 3]
```diff
@@ -170,6 +170,8 @@ workflows:
           filters:
             branches:
               only:
+                # FIXME:
```
Comment (maudetes) [inline .circleci/config.yml:173]: Don't forget the FIXME :)

**Review by maudetes — APPROVED** [review]

**Pull request merged**